scripts/prepare_sector_network.py

```diff
@@ -2254,6 +2579,47 @@ def add_heat(n: pypsa.Network, costs: pd.DataFrame, cop: xr.DataArray):
                 )
 
 
+def add_methanol(n, costs):
```

Comment:
can one adapt the function `def add_carrier_buses`[here](https://github.com/PyPSA/pypsa-eur/blob/05dba2d36b5a9b640bbb659d627f21f7ae15b5a1/scripts/prepare_sector_network.py#L523) to work with methanol as well?

Reply:
I previously thought the same but I decided against it. However, now I do not see any reason to not use ``add_carrier_busses`` so I used it again.